sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/response/RefundDetailsResponse.java

```diff
@@ -0,0 +1,117 @@
+package nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.response;
+
+import org.json.JSONObject;
+
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.Money;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Response from the Rabobank API when a refund is created
+ */
+public class RefundDetailsResponse {
+
+    private final UUID refundId;
+    private final UUID refundTransactionId;
+    private final String createdAt;
```

Comment:
Could maybe be converted to a datetime?

Reply:
I would leave it as a String for now. Appears that API is serializing OffsetDateTime in a non-parsable format (eg "2000-01-01T00:00:00.000-0200", the : sign on offset is missing) so that can cause some confusion.

Reply:
agreed